librz/arch/isa/pic/pic18_analysis.inc

```diff
@@ -19,7 +19,7 @@ typedef struct {
 typedef struct {
 	const Pic18Op *op;
 	const HtSU *mm;
-	ILOpEff eff;
+	RzVector /*<ILOpEff>*/ effs;
```

Comment:
why no just use seq? they are one in each other. check their implementation

Reply:
Because Seq is a series of side effects, or statements, it cannot be used as a value.

Some registers in the PIC18 trigger additional side effects when accessed, such as +1 after the access, +1 before the access, etc.

To be able to hook specific register accesses in SETG and VARG and add some side effects I had to come up with this method.